apps/web/src/components/Twitter/TwitterFollowingModal.tsx

```diff
@@ -0,0 +1,202 @@
+import Link from 'next/link';
+import { useCallback, useMemo } from 'react';
+import { graphql, useFragment, usePaginationFragment } from 'react-relay';
+import styled from 'styled-components';
+
+import { useReportError } from '~/contexts/errorReporting/ErrorReportingContext';
+import { useModalActions } from '~/contexts/modal/ModalContext';
+import { TwitterFollowingModalFragment$key } from '~/generated/TwitterFollowingModalFragment.graphql';
+import { TwitterFollowingModalMutation } from '~/generated/TwitterFollowingModalMutation.graphql';
+import { TwitterFollowingModalQueryFragment$key } from '~/generated/TwitterFollowingModalQueryFragment.graphql';
+import { usePromisifiedMutation } from '~/hooks/usePromisifiedMutation';
+import { removeNullValues } from '~/shared/relay/removeNullValues';
+
+import breakpoints from '../core/breakpoints';
+import { Button } from '../core/Button/Button';
+import Markdown from '../core/Markdown/Markdown';
+import { HStack, VStack } from '../core/Spacer/Stack';
+import { BaseM, TitleDiatypeL } from '../core/Text/Text';
+import FollowButton from '../Follow/FollowButton';
+
+type Props = {
+  followingRef: TwitterFollowingModalFragment$key;
+  queryRef: TwitterFollowingModalQueryFragment$key;
+};
+
+export default function TwitterFollowingModal({ followingRef, queryRef }: Props) {
+  const query = useFragment(
+    graphql`
+      fragment TwitterFollowingModalQueryFragment on Query {
+        ...FollowButtonQueryFragment
+      }
+    `,
+    queryRef
+  );
+
+  const followingPagination = usePaginationFragment(
+    graphql`
+      fragment TwitterFollowingModalFragment on Query
+      @refetchable(queryName: "TwitterFollowingModalQuery") {
+        socialConnections(
+          before: $twitterListBefore
+          last: $twitterListLast
+          socialAccountType: Twitter
+        ) @connection(key: "TwitterFollowingModal__socialConnections") {
+          edges {
+            node {
+              __typename
+              ... on SocialConnection {
+                __typename
+                galleryUser {
+                  ... on GalleryUser {
+                    username
+                    bio
+                    ...FollowButtonUserFragment
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    `,
+    followingRef
+  );
+
+  const { hideModal } = useModalActions();
+
+  const handleClose = useCallback(() => {
+    hideModal();
+  }, [hideModal]);
+
+  const twitterFollowing = useMemo(() => {
+    const users = [];
+
+    for (const edge of followingPagination?.data?.socialConnections?.edges ?? []) {
+      if (edge?.node?.__typename === 'SocialConnection') {
+        users.push(edge?.node.galleryUser);
+      }
+    }
+
+    return removeNullValues(users);
```

Comment:
You can save this by doing `if(edge?.node.galleryUser)` above

Reply:
Done!